index.js

```diff
@@ -1,8 +1,10 @@
 const express = require('express'),
       morgan = require('morgan'),
       bodyParser = require('body-parser'),
-      pug = require('pug'),
-      fs = require('fs');
+      fs = require('fs'),
+      pug = require('pug');
+
+var likeCount = JSON.parse(fs.readFileSync('likes.json')).likes;
```

Comment:
it is our likeCount cache for the app, we are writing the changed count to the likes.json during the /likes post handler. Because of the way our json is structured in order to get the likeCount we do .likes property/value lookup on the parsed json object, check likes.json